### ISSUE TITLE
Feature request: Send location on Telegram per user

### ISSUE THREAD

**Issue by DanielHoelder** [traccar/traccar#5174]
Currently, the parameter notificator.telegram.sendLocation can only be set for the whole server in the config file (traccar.xml). 

It would be useful if it was possible to do that user specific as an attribute in the user config instead of the config file.

**Comment by 0xenca** [comment]
This can be done by setting in the user attributes.

![image](https://github.com/traccar/traccar/assets/106509878/de555d80-1b6b-440a-aa5e-f21d00255d14)

**Comment by tananaev** [comment]
No, it cannot. @0xenca I think you didn't read the description carefully enough.
